### PR TITLE
Polish augments

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Add scrolling hacking material on fire for hacking extension screen #321 
- Add and change some wanted scoreboard phrases
- Add antenna wobble to seeker augment
- Add radar to seeker augment #338 
- Add more lerp to rubber sniper, fix momentum bending being in the correct coordinate system
- Add new and better jiggle to augment wires
sekkerRadar

### CODE REVIEW

**Review by toberge — CHANGES_REQUESTED** [review]
Two things:
- The radar turns very abruptly when it has a new target, this should ideally be lerped
- The hacking extension is kinda tiny when it's all the way on the edge of the gun... but that's not something we can fix when we have such lomg parts. It does make me wish we had some alternate placements for extensions.
image

File: Assets/Scripts/Augment/AugmentImplementations/AmmoBox.cs [hunk 1]
```diff
@@ -24,6 +26,23 @@ private void Start()
...
+    {
+        ammoBoxes.Add(this);
+    }
+
+    private void OnDisable()
+    {
+        ammoBoxes.Remove(this);
+    }
+
+    public static AmmoBox GetClosestAmmoBox(Transform from)
```
Comment (toberge) [inline Assets/Scripts/Augment/AugmentImplementations/AmmoBox.cs:39]: This method could take a position vector, it doesn't use anything but `from.position`

Comment (Fueredoriku) [reply Assets/Scripts/Augment/AugmentImplementations/AmmoBox.cs]: Done 👌

**Review by toberge — APPROVED** [review]
Nice upgrades!

**Pull request merged**